js/sign.js

```diff
@@ -2,100 +2,200 @@ const useremail = document.querySelector('#useremail')
 const signInputEmail = document.querySelector('.sign-input-email');
 
 // 이메일 타입이 잘못되었는지 확인하는 함수
-function emailTypeChacking (useremail){
+function checkEmailType (useremail){
   const exptext = /^([0-9a-zA-Z_\.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
   if(exptext.test(useremail) === false){
   //이메일 형식이 알파벳+숫자@알파벳+숫자.알파벳+숫자 형식이 아닐경우			
-  errPaint('올바른 이메일 주소가 아닙니다', signInputEmail)
+  paintErr('올바른 이메일 주소가 아닙니다', signInputEmail)
   return false;
   }
 }
 
 // 에러문구 표기 함수
-function errPaint(massage, append){
+function paintErr(message, parent){
   const p = document.createElement('p');
-  p.innerText = massage;
+  p.innerText = message;
   p.classList.add('input-err');
-  append.appendChild(p);
+  parent.appendChild(p);
+}
+
+function paintErrLine (inputTag){
+  if (inputTag.classList.contains('errLine')){
+    return;
+  } else{
+    inputTag.classList.add('errLine');
+  }
+}
+
+function removeErrLine (inputTag){
+  if (inputTag.classList.contains('errLine')){
+    inputTag.classList.remove('errLine');
+  } else{
+    return;
+  }
 }
```

Comment:
if-else 문보다 if 문으로 더 간결하게 표현해도 좋을 것 같아요.

```suggestion
function paintErrLine (inputTag){
  const hasErrorLine = inputTag.classList.contains('errLine')
  
  if (!hasErrorLine) {
    inputTag.classList.add('errLine');
  }
}

function removeErrLine (inputTag){
  const hasErrorLine = inputTag.classList.contains('errLine')
  
  if (hasErrorLine) {
    inputTag.classList.remove('errLine');
  }
}
```